client/allocdir/alloc_dir.go

```diff
@@ -28,6 +28,15 @@ const (
 	// idUnsupported is what the uid/gid will be set to on platforms (eg
 	// Windows) that don't support integer ownership identifiers.
 	idUnsupported = -1
+
+	// fileMode777 is a constant that represents the file mode rwxrwxrwx
+	fileMode777 = os.FileMode(0o777)
+
+	// fileMode755 is a constant that represents the file mode rwxr-xr-x
+	fileMode755 = os.FileMode(0o755)
+
+	// fileMode644 is a constant that represents the file mode rw-r--r--
```

Comment:
The comment doesn't match the implementation:

```suggestion
	// fileMode666 is a constant that represents the file mode rw-rw-rw-
```